Lib/test/string_tests.py

```diff
@@ -132,15 +144,7 @@ def test_count(self):
         #    replacing all occurrences and then differencing the string lengths
         charset = ['', 'a', 'b']
         digits = 7
-        base = len(charset)
-        teststrings = set()
-        for i in range(base ** digits):
-            entry = []
-            for j in range(digits):
-                i, m = divmod(i, base)
-                entry.append(charset[m])
-            teststrings.add(''.join(entry))
-        teststrings = [self.fixtype(ts) for ts in teststrings]
+        teststrings = self._get_teststrings(charset, digits)
```

Comment:
`charset` and `digits` can be inlined:
```suggestion
        teststrings = self._get_teststrings(['', 'a', 'b'], 7)
```